resolwe_bio/tests/processes/test_junctions.py

```diff
@@ -76,28 +76,28 @@ def test_no_sj_inputs(self):
                                                               source='ENSEMBL',
                                                               species='Homo sapiens',
                                                               build='GRCh38_ens90')
-            inputs = {
+
+            genome_no_junctions = self.run_process('upload-fasta-nucl', {
                 'src': 'HS chr21_ensembl.fa.gz',
                 'species': 'Homo sapiens',
                 'build': 'GRCh38_ens90',
-            }
-            genome_no_junctions = self.run_process('upload-genome', inputs)
+            })
 
-            inputs = {'annotation': annotation_no_junctions.id, 'genome': genome_no_junctions.id}
-            star_index_no_junctions = self.run_process('alignment-star-index', inputs)
+            star_index_no_junctions = self.run_process('alignment-star-index', {
+                'annotation': annotation_no_junctions.id,
+                'ref_seq': genome_no_junctions.id
+            })
 
-            inputs = {
+            star_no_junctions = self.run_process('alignment-star', {
                 'genome': star_index_no_junctions.id,
                 'reads': reads_no_junctions.id,
-            }
-            star_no_junctions = self.run_process('alignment-star', inputs)
+            })
 
-            inputs = {
+            empty_bam_upload = self.run_process('upload-bam', {
```

Comment:
It would be best to put such formatting in a separate commit, but IMO your time is better spent on other stuff than making this "right". Just for future reference. :-)